ui/src/layouts/chats/presentation/messages/mod.rs

```diff
@@ -69,6 +69,8 @@ pub enum MessagesCommand {
 }
 
 pub type DownloadTracker = HashMap<Uuid, HashSet<warp::constellation::file::File>>;
+const READ_SCROLL: &str = "return document.getElementById(\"messages\").scrollTop";
```

Comment:
I guess this is fine. Though if it was moved into the `chats/scripts/mod.rs` I think that would help us keep track of what scripts are being used.